package.json

```diff
@@ -20,6 +21,7 @@
     "immutable": "3.x.x"
   },
   "devDependencies": {
+    "@types/draft-js": "^0.7.33",
```

Comment:
Why is it necessary to add this dev dependency? It's not being used anywhere in the code as far as I can tell. @dimafeng any thoughts?